system/modules/playos-status.nix

```diff
@@ -0,0 +1,57 @@
+{ config, lib, pkgs, ... }:
+let
+  ttyNumber = 8;
+  tty = "tty${toString ttyNumber}";
+  ttyPath = "/dev/${tty}";
+  driverUrl = "http://127.0.0.1:8382";
+
+  script =
+    pkgs.writeShellScriptBin "print-status" ''
+      while :; do
+        screen=$(xrandr --current | grep '*' | awk '{print $1}')
+        networkCount=$(connmanctl services | grep wifi | wc -l)
+        networkConnected=$(connmanctl services | grep wifi | grep "*" | awk '{ print $2 }')
+        rfid=$(curl -s "${driverUrl}/rfid/readers" | jq -r ".readers" | jq length)
```

Comment:
I realized that this will not work. The reason is that the driver only looks for readers when there is a WebSocket listener.

A simple solution might be [`pcsc_scan`](http://ludovic.rousseau.free.fr/softwares/pcsc-tools/) which is available through `pcsctools` in Nixpkgs.

The output of `pcsc_scan` looks like this:

```
❖ pcsc_scan -r
Using reader plug'n play mechanism
Scanning present readers...
0: Yubico YubiKey OTP+FIDO+CCID 00 00
1: Alcor Micro AU9560 01 00
```

Reply:
I ran into an issue with this. `pcsc_scan -r` doesn't return if there are no readers connected:
```
$ pcsc_scan -r
Using reader plug'n play mechanism
Scanning present readers...
Waiting for the first reader...
```

I found [`opensc-tool`](https://htmlpreview.github.io/?https://github.com/OpenSC/OpenSC/blob/master/doc/tools/tools.html#opensc-tool), which could be used with the `--list-readers` option. What do you think?

Reply:
> I found [`opensc-tool`](https://htmlpreview.github.io/?https://github.com/OpenSC/OpenSC/blob/master/doc/tools/tools.html#opensc-tool), which could be used with the `--list-readers` option. What do you think?

I am not familiar with it, but looks fine at a quick glance.

Reply:
Fixed this using `opensc-tool`.